templates/index.html

```diff
@@ -43,6 +43,20 @@ <h2>The app store for Linux</h2>
   </div>
 </div>
 
+<div class="p-strip">
+    <div class="row">
```

Comment:
Seems like indentation got a bit shifted after copy/pasting.

Reply:
Sorry about that. Fixed as 81c16b1